pkg/build/controller/strategy/custom.go

```diff
@@ -104,13 +107,33 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildv1.Build, additionalCA
 					Env:                      containerEnv,
 					SecurityContext:          securityContext,
 					TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+					VolumeMounts: []corev1.VolumeMount{{
+						Name:      BuildWorkDirVolume,
+						MountPath: buildutil.BuildWorkDirMount,
+					}},
+					// setting the container to run directly in the location the repository is cloned
+					// using the openshift helper
+					WorkingDir: workingDir,
```

Comment:
Note to self: cri-o creates the container's working directory if necessary.

Reply:
That's good to know, thanks for sharing!

Reply:
Oh, the chmod container probably needs this set, too, so that the runtime will ensure the directory exists.  If the git-clone container doesn't create the directory, or it isn't used, the runtime won't know to create it for the chmod container otherwise.

Reply:
Indeed, the `git-chmod` container definitely needs that setting, thanks for the heads up!